src/components/card/z-card-footer/index.tsx

```diff
@@ -26,38 +24,74 @@ export class ZCardFooter {
 
   @State() isOpen: boolean = false;
 
+  @State() allowTooltipAuthors: boolean = false;
+
+  private footer?: HTMLElement;
+  private ellipsisAuthors?: HTMLElement;
+
   @Listen("toggleClick")
   handleToggle(): void {
     this.isOpen = !this.isOpen;
   }
 
+  getTitleAuthors(): string {
+    return this.allowTooltipAuthors ? this.autori : "";
+  }
+
+  componentDidLoad() {
+    this.footer.ontransitionend = event => {
```

Comment:
Why programmatically adding the event listener instead of using the `@Listen` decorator of Stencil?

Reply:
Otherwise, instead of using the ref I think it would be better to inline the listener in the template.